app/mailers/user_mailer.rb

```diff
@@ -2,7 +2,8 @@ class UserMailer < ActionMailer::Base
   include Mailable
   include LocaleHelper
   before_action :attach_images
-  default from: email_with_name(Figaro.env.email_from, Figaro.env.email_from)
+  default from: email_with_name(Figaro.env.email_from, Figaro.env.email_from),
+          reply_to: email_with_name(Figaro.env.email_from, Figaro.env.email_from)
```

Comment:
AFAIK the `Reply-To` header isn't useful unless you want replies to be sent somewhere other than the `From` header.

Reply:
That was actually added for the sake of consistency. As mentioned, our small group of generated emails follows one of two paths: either via our own Rails mailers, or Devise-generated messages. (Both ultimately call `mail()` in Rails.)

Devise messages include the header by default, and since we're no longer customizing those, it made sense to add it to our own mailers as well, with no discernible cost (and possibly a small boost) to deliverability. Unless we don't like the semantics of a non-reply `Reply-To`, in which case we would need to again customize the headers. (I'd advise not.)